can-view-parser.js

```diff
@@ -270,6 +271,12 @@ var callAttrStart = function(state, curIndex, handler, rest){
 		dev.warn("can-view-parser: Found attribute with name: ", oldAttrName, ". Converting to: ", newAttrName);
 		//!steal-remove-end
 	}
+
+	//encode spaces
+	if(spacesRegex.test(attrName)){
```

Comment:
You mean just always run the replace and get rid of the `if` statement?

```js
newAttrName = attrName.replace(spacesRegex, "\\s");
```
instead of 
```js
	if(spacesRegex.test(attrName)){
		newAttrName = attrName.replace(spacesRegex, "\\s");
	}
```